dgraphtest/dgraph.go

```diff
@@ -102,14 +102,31 @@ func (z *zero) workingDir() string {
 }
 
 func (z *zero) mounts(conf ClusterConfig) ([]mount.Mount, error) {
-	return []mount.Mount{
-		{
-			Type:     mount.TypeBind,
-			Source:   os.Getenv("GOPATH") + "/bin",
-			Target:   "/gobin",
-			ReadOnly: true,
-		},
-	}, nil
+
+	if conf.isUpgrade {
```

Comment:
we should decide this based on the current version of dgraph that the cluster is running